pkg/cli/predict.go

```diff
@@ -204,11 +160,11 @@ func predictIndividualInputs(ui terminal.UI, deployment serving.Deployment, inpu
 		return err
 	}
 
-	ui.Output("Written output to " + outputPath)
+	console.Infof("Written output to %s", outputPath)
 	return nil
 }
 
-func parsePredictInputs(inputs []string) *serving.Example {
+func parsePredictInputs(inputs []string) predict.Input {
```

Comment:
Hmm, yes. I was trying to avoid adding a new abstraction here, because we call them "inputs" in most other places. Just `predict.Inputs` maybe? And the function could be called `parseInputFlags` or something to make it clear what those strings are.

There is `image.Input` too which is a singular input. But that represents a type, not an instance of an input, so perhaps that should be renamed `InputType` for clarity.